src/templates/press.js

```diff
@@ -1,61 +1,79 @@
+import '../sass/styles.scss';
 import React from 'react';
-import { graphql } from 'gatsby';
-import ReactMarkdown from "react-markdown";
+import {graphql, Link, useStaticQuery} from 'gatsby';
 import RedLayout from '../layouts/red';
+import Hero from '../components/hero';
+import SEO from '../components/seo';
 import PrimaryPageCTA from "../components/primary-page-cta";
-import SEO from "../components/seo";
-import Hero from "../components/hero";
+import PressReleaseGrid from '../components/press-release-grid.js';
+import Pagination from "../components/pagination.js";
 
-const PressTemplate = ({ data }) => {
-  const press = data.allStrapiPressRelease.edges[0].node;
-  const { Title, Body } = press;
+
+const Press = ({ data, location, pageContext }) => {
+  const pressReleases = data.press.edges;
   return (
     <RedLayout>
-
       <SEO
-        title={Title}
-        description='Press release from civicactions.com'
+        title='Press'
+        description="This work is worth talking about. Learn how we've been elevating government digital services."
       />
-      <Hero title={Title}/>
-      <div className='single--press-releases--main'>
+      <Hero
+        title='This work is worth talking about.'
+        description='When government invests in smart technology and thoughtful design of services, millions of people stand to benefit for years to come. We are honored to be part of this work.'
+      />
+      <section className='press-releases'>
+        <div className='inner'>
 
+          {pressReleases.map(({ node }) => {
+              return (
+                <PressTeasers presses={node}/>
+                )
 
-        <div className='inner'>
-          <div className='body'>
-            <ReactMarkdown className='body' children={Body} />
-          </div>
+
+          })}
+          <Pagination pageContext={pageContext} />
         </div>
-      </div>
+
+      </section>
       <PrimaryPageCTA
-        title='Inspired by newsworthy work?'
-        subtitle='See what we can do together.'
-        primaryButtonText='EXPLORE SERVICES'
-        primaryButtonLink='/services'
-        secondaryButtonText='CONTRACTING INFO'
-        secondaryButtonLink='/contracting'
+          title='Inspired by newsworthy work?'
+          subtitle='See what we can do together.'
+          primaryButtonText='EXPLORE SERVICES'
+          primaryButtonLink='/services'
+          secondaryButtonText='CONTRACTING INFO'
+          secondaryButtonLink='/contracting'
       />
-
-
-
     </RedLayout>
   );
 };
 
-export const query = graphql`
-query PressQuery($pagePath: String!) {
-  allStrapiPressRelease(filter: { Path: { eq: $pagePath } }) {
-    edges {
-      node {
-        Title
-        Body
-        Date
-        Path
-      }
+const PressTeasers = ({ presses }) => {
```

Comment:
This is a total nitpick, so take it or leave it but it feels like this doesn't need to be abstracted out into a component like this since it is just returning a one dimensional component. You could just use the `PressReleaseGrid` component in place of `PressTeasers`

Reply:
Good thinking. Fixed.